mapie/classification.py

```diff
@@ -208,6 +209,7 @@ def __init__(
         cv: Optional[Union[int, str, BaseCrossValidator]] = None,
         test_size: Optional[Union[int, float]] = None,
         n_jobs: Optional[int] = None,
+        conformity_score: Optional[BaseClassificationScore] = None,
```

Comment:
default conformity_score should be LAC() and defautl method should be None in order to get used to define the conformity_score and not the method

Reply:
We should also change their order

Reply:
To ensure that notebooks do not break, we cannot change the order (in the case where the user instantiates an object with the attributes but without mentioning the keys).

Reply:
Thank you for the first comment, which I will take into account.

Reply:
I'm having issues with sklearn's `check_estimator.`
- I had to do like MapieRegressor for which `conformity_score=None` by default.
- This doesn't change `conformity_score` to take precedence over `method`.
- This time, `method=None` by default.

The consequence is that I had to delete a test which was no longer useful with more default values ​​other than None.

Reply:
Done 👍